pom.xml

```diff
@@ -845,5 +841,20 @@ Import-Package: \\
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>j17</id>
+      <activation>
+        <jdk>[17.0,18.0);[21.0,22.0)</jdk>
+      </activation>
+      <properties>
+        <maven.compiler.release>17</maven.compiler.release>
+      </properties>
+    </profile>
```

Comment:
Is this profile necessary when `<maven.compiler.release>17</maven.compiler.release>` would be as default value? Maybe acttivating the j21 profile would then simply override it?